docs/optimizing/advanced_retrieval/How to get +15% hit_rate improvement using DeepMemory.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.            relevance.append([(relevant_docs[doc][0], 1)])
</pre><p>so relevance here is fixed as 1.</p><p>is it possible to send graded relevance</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
not yet